README.md

```diff
@@ -16,7 +16,7 @@ with CQC tket, a set of quantum programming tools.
 `pytket-qujax` is an extension to `pytket` that allows `pytket` circuits to
 be converted to `qujax`.
 
-[qujax](https://cqcl.github.io/qujax/api/) is a pure [JAX](https://github.com/google/jax)
+[qujax](https://github.com/CQCL/pytket-qujax) is a pure [JAX](https://github.com/google/jax)
```

Comment:
Is this change intended?

Reply:
I meant to change it to the github link (I think it's better for github to link to github)
but mistakenly linked it to pytket-qujax rather than qujax lol